src/_includes/layouts/base.njk

```diff
@@ -1,3 +1,25 @@
+---
+meta:
+  language: en-US
+  site:
+    name: FlowFuse
+    description: Bespoke, flexible, and resilient manufacturing applications
+    url: https://FlowFuse.com
+    logo:
+      src: https://flowfuse.com/handbook/images/logos/ff-logo--square--dark.png
```

Comment:
@Yndira-FlowForge If you can find a better one, that would be ideal

Reply:
Do you know where it is going to be displayed? Just to see if the squared one is better, or if the background will be light or dark